sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/InstallerUtils.java

```diff
@@ -53,6 +55,22 @@
         /* Hide constructor in utils pattern. */
     }
 
+    /**
+     * Get the intent used to open installation U.I.
+     *
+     * @param fileUri downloaded file URI from the download manager.
+     * @return intent to open installation U.I.
+     */
+    @NonNull
+    public static Intent getInstallIntent(Uri fileUri) {
+        Intent intent = new Intent(Intent.ACTION_INSTALL_PACKAGE);
+        intent.setData(fileUri);
+        intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
```

Comment:
Is this a use case? Starting install intent already having it started?

Reply:
It isn't a new code. Let's do not change this logic